### PR TITLE
add dockerfile

### LINKED ISSUE #3: Add a dockerfile for this API

### PR DESCRIPTION
**Pull request by trevormccasland**

### CODE REVIEW

**Review by radiomistress — APPROVED** [review]

**Review by radiomistress — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+FROM python:3.9
+
+WORKDIR /app
+
+COPY . /app
+
+RUN pip install -r /app/requirements.txt
+
+COPY ./dotalytics_api /app/dotalytics_api
```
Comment (stannum-l) [inline Dockerfile]: isn't this a dup of L5?

Comment (stannum-l) [reply Dockerfile]: if the WORKDIR is set to /app do you need to specify /app everywhere?

Comment (trevormccasland) [reply Dockerfile]: Yes, you're right. Whoops

Comment (trevormccasland) [reply Dockerfile]: I removed the /app specification, thanks for the review!

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,9 @@
+FROM python:3.9
```
Comment (stannum-l) [inline Dockerfile]: Should use 3.10

**Pull request merged**